json-rpc/eth_getBalance.md

```diff
@@ -0,0 +1,17 @@
+# `Eth_getBalance`
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+Specification | Description 
+---|---
+1|Eth_getBalance must be sent a valid Ethereum address as the first parameter|
```

Comment:
I think this is not true.

Reply:
If you mean formatting, this should be covered in a metadocument for the address type